api_tests/.gitignore

```
@@ -3,3 +3,4 @@ log
 *.js
 gen/
 yarn-error.log
+dist/*
```

```suggestion
dist/
```
So that we do not check-out/in an empty folder.